routes/goldap.js

```diff
@@ -215,12 +274,25 @@ module.exports = {
             logger.error(e);
             throw e;
         }
-        
+
         try {
             let created_file = await filer.ldap_add_user(user, group, fid);
             logger.debug('File created', created_file);
-            created_file = await filer.ldap_add_user_to_group(user, fid);
-            logger.debug('File created', created_file);
+
+            await get_group_dn(group.name)
+                .then(
+                    group_dn => { // resolve()
+                        return filer.ldap_add_user_to_group(user,group_dn, fid);
+                    })
+                .then(
+                    created_file => {
+                        logger.info('File Created: ', created_file);
+                    })
+                .catch(error => { // reject()
+                    logger.error('Delete Group Failed for: ' + group.name, error);
```

Comment:
Wrong message, here we add user to group

Reply:
a9bcd2bf89ae0097a98746aa28f8b2f75d330005